### PR TITLE
build examples

### PR DESCRIPTION
**Pull request by bsima**
- Don't use stdenv.lib
- Build examples with nix

### CODE REVIEW

File: examples/default.nix [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{stdenv, mkDerivation, lib, base, clay, text}:
```
Comment (turion) [inline examples/default.nix:1]: Rather, can you use `callCabal2nix`?

**Review by turion — APPROVED** [review]
Thanks for the improvement! Just a small suggestion, if you want.

**Comment by bsima** [comment]
Manuel Bärenz ***@***.***> writes:

> Rather, can you use `callCabal2nix`?

Done.

**Comment by turion** [comment]
Awesome, thanks!

**Pull request merged**